public/i18nUtils.js

```diff
@@ -0,0 +1,29 @@
+import getCountry from './getCountry.js';
+
+export const DEFAULT_LOCALE = 'en';
+
+// default is null so that i18n can fallback to browser locale preferences
+// fallback to en if no locale detected is handled there
```

Comment:
Currently only calling this method in two places, and passing DEFAULT_LOCALE both times. Planning for future browser-based detection?

Reply:
I'll do a little refactoring in i18n.js to make a bit clearer but we do need it there because we need to fallback to the jquery.i18n detected locale and then 'en'